scripts/main.js

```diff
@@ -0,0 +1,137 @@
+// 1.1
+var nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+function getEvenNums1(arr) {
+  if (!Array.isArray(arr)) throw new TypeError;
+
+  let evNums1 = [];
+  for (let i = arr.length - 1; i >= 0; i--) {
```

Comment:
it's always better to start from the beginning of the array

Reply:
It was just as example of another use of loop and I know (maybe it's not true) that if you have a lot of date it would be better to iterate from the and - but it's only in case of really big data.
Fixed